tests/datasets/test_groceriesreal.py

```diff
@@ -1,107 +0,0 @@
-import os
```

Comment:
@adason  Should i add back in this test?

Reply:
Yes. The changes in this PR only focus on `DatasetDownloader` interface and `UnitySimulationDownloader` and does not make changes to groceriesreal dataset.